common/src/main/java/com/netflix/conductor/common/metadata/tasks/TaskDef.java

```diff
@@ -142,11 +146,17 @@ public TaskDef(String name, String description) {
         this.description = description;
     }
 
-    public TaskDef(String name, String description, int retryCount, long timeoutSeconds) {
+    public TaskDef(
```

Comment:
Can a new constructor be added instead of changing the signature of this one?

Reply:
+1